platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/reconnect/LearnerPullVirtualTreeView.java

```diff
@@ -132,12 +134,14 @@ public LearnerPullVirtualTreeView(
             final VirtualStateAccessor originalState,
             final VirtualStateAccessor reconnectState,
             final ReconnectNodeRemover<K, V> nodeRemover,
-            final NodeTraversalOrder traversalOrder) {
+            final NodeTraversalOrder traversalOrder,
+            final ReconnectMapStats mapStats) {
```

Comment:
missing doc and nullity annotation for new parameter

Reply:
added